pkg/controller/util/placement.go

```diff
@@ -20,11 +20,13 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
+const clusterNamesField = "clusterNames"
+
 func GetClusterNames(placement *unstructured.Unstructured) ([]string, error) {
-	clusterNames, _, err := unstructured.NestedStringSlice(placement.Object, "spec", "clusternames")
+	clusterNames, _, err := unstructured.NestedStringSlice(placement.Object, "spec", clusterNamesField)
```

Comment:
(no action required) we may want to consider returning the boolean indicating whether the value of the field was found. The caller can choose to do what it wants with it. In the case of the crudtester, it can emit a fatal error if it's expecting a value for this field to be there and it's not.